docs/next/components/JoinSlackButton.tsx

```diff
@@ -1,37 +1,26 @@
-import { useRouter } from "next/router";
 import React from "react";
 
-export const JoinSlackButton: React.FC<{ header?: boolean }> = ({ header }) => {
+export const JoinSlackButton: React.FC<{}> = () => {
   const [open, setOpen] = React.useState(false);
-  const { query } = useRouter();
-
-  React.useEffect(() => {
-    if (header && query.slack) {
-      setOpen(true);
-    }
-  }, [query.slack]);
 
   return (
     <>
-      <div
-        onClick={() => setOpen(false)}
-        style={{ marginLeft: 0 }}
-        className={`h-screen inset-0 fixed z-50 flex flex-col justify-center items-center bg-opacity-25 bg-black transition-opacity
-        ${open ? "opacity-1" : "opacity-0 pointer-events-none"} `}
```

Comment:
the flashing and the white box seem to be caused by `opacity-0` here because when the modal is not open, it would try load it and then set it to opacity-0.